### PR TITLE
Updates .Net on TfsInterop.VS2015 for future Support of System.Identi…

### PR DESCRIPTION
**Pull request by CDAGaming**
…tyModel.Tokens.Jwt(Now on V5.1.3)

Signed-off-by: CDAGaming <cstack2011@yahoo.com>

Changes proposed in this pull request:
 - In Title

How did I test this code:
 - Used in Daily Enviroment

Has been tested on (remove any that don't apply):
 - GIT 2.10 and above
 - Windows 7 and above

### CODE REVIEW

**Comment by CDAGaming** [comment]
IMPORTANT: If necessary, I can take the time this weekend to migrate the entirety of git extensions to .net 4.6.2

**Comment by CDAGaming** [comment]
Really Appveyor?! Why do i need to babysit it :P

File: Plugins/BuildServerIntegration/TfsInterop.Vs2015/TfsInterop.Vs2015.csproj [hunk 1]
```diff
@@ -109,8 +115,8 @@
     </Reference>
     <Reference Include="System" />
     <Reference Include="System.Core" />
-    <Reference Include="System.IdentityModel.Tokens.Jwt, Version=4.0.40306.1554, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\..\..\packages\System.IdentityModel.Tokens.Jwt.4.0.4.403061554\lib\net45\System.IdentityModel.Tokens.Jwt.dll</HintPath>
+    <Reference Include="System.IdentityModel.Tokens.Jwt, Version=5.1.3.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <HintPath>..\..\..\packages\System.IdentityModel.Tokens.Jwt.5.1.3\lib\net451\System.IdentityModel.Tokens.Jwt.dll</HintPath>
```
Comment (RussKie) [inline Plugins/BuildServerIntegration/TfsInterop.Vs2015/TfsInterop.Vs2015.csproj]: shouldn't it be `...\lib\net462\..`?

Comment (CDAGaming) [inline Plugins/BuildServerIntegration/TfsInterop.Vs2015/TfsInterop.Vs2015.csproj]: Not sure. I think it's targeting highest library which is 4.5.1

Comment (CDAGaming) [inline Plugins/BuildServerIntegration/TfsInterop.Vs2015/TfsInterop.Vs2015.csproj]: Not sure. I think it's targeting highest library which is 4.5.1

Comment (CDAGaming) [inline Plugins/BuildServerIntegration/TfsInterop.Vs2015/TfsInterop.Vs2015.csproj]: Not sure. I think it's targeting highest library which is 4.5.1

Comment (CDAGaming) [inline Plugins/BuildServerIntegration/TfsInterop.Vs2015/TfsInterop.Vs2015.csproj]: Ah, Cruddy At&t Service. Apologies if you received duped message replies

**Comment by CDAGaming** [comment]
@RussKie Going to close this for the day while syncing your Fork with master, and using it. Next Time, ill set the Framework down to 4.5.2 for this

**Pull request closed without merging**